### DISCUSSION TITLE
UCD Log - Service Design

### DISCUSSION THREAD

**Discussion by anabrighenti** [nhsbsa/nhsbsa-digital-playbook#243] [UCD]
The SSDs have worked on [guidance for writing a service design log](https://nhsbsauk-my.sharepoint.com/:w:/g/personal/jansp_nhsbsa_nhs_uk/ETtCZ780WbNDqBsDDdx51VoBHt9nlD-VeObw7IvV6PFsZw?e=dZPsOW): 

The purpose of the UCD log is to provide documentation and explanation on how and why decisions were made within a service. This will be tested and reviewed accordingly to keep iterating and improving our processes.

**Comment by nhsbsa-deaco** [comment]
Email has been sent out to all service designers to collate feedback on the guide

**Reply by NettySP8** [reply]
Also awaiting the UR feedback outcomes on the workshop with the design community.

**Reply by nhsbsa-deaco** [reply]
No more feedback has been given from the design community on the Interaction and Service Design guidance content.

I think it going to be a case of just moving forward with the guidance we have.

**Reply by NettySP8** [reply]
The workshop infos due back this week so we can inform all design (ID,SD and interaction) guidance if anything comes out of it

**Reply by nhsbsa-deaco** [reply]
just a note from discussions outside of this, findings are being collated and shared soon

**Comment by nhsbsa-sthil** [comment]
Just adding the link to the Service Design guidance for visibility:

https://nhsbsauk-my.sharepoint.com/:w:/g/personal/jansp_nhsbsa_nhs_uk/ETtCZ780WbNDqBsDDdx51VoBHt9nlD-VeObw7IvV6PFsZw?e=R3P5Kf

**Reply by NettySP8** [reply]
Great thanks Stephen

**Comment by nhsbsa-deaco** [comment]
I will share the feedback on the IxD/SD guidance here shortly

**Reply by nhsbsa-deaco** [reply]
In total we have received 7 responses to the feedback form.

![Interaction Service Design Guidance Feedback](https://github.com/nhsbsa/nhsbsa-digital-playbook/assets/51478242/9c4c4518-79fa-4403-a989-ae55e4c206d6)

Excel document of the feedback:
[Draft Guide for Design Logs (Feedback)(1-7) (1).xlsx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14053998/Draft.Guide.for.Design.Logs.Feedback.1-7.1.xlsx)

**Comment by nhsbsa-deaco** [comment]
Service Design UCD Log Guidance is now on the playbook in DRAFT status, so needs to be accessed via a direct link.

Link: https://nhsbsa.github.io/nhsbsa-digital-playbook/design/service-designer/ucd-log/

**Comment by nhsbsa-deaco** [comment]
Service Design Blank Template - Ready for Pilot

[Service Design UCD Log.docx](https://github.com/nhsbsa/nhsbsa-digital-playbook/files/14534810/Service.Design.UCD.Log.docx)
